src/method_nlar.jl

```diff
@@ -194,19 +129,21 @@ function nlar(::Type{T},
 
             #Check for convergence of one of the eigenvalues
             err = errmeasure(nu,u);
-            println(k," Error:",err," Eigval :",nu)
+            if(displaylevel == 1)
+                println(k," Error:",err," Eigval :",nu)
+            end
             err_hyst[k,m+1]=err;    # Giampaolo's edit
```

Comment:
err_hyst? Shouldnt it be err_hist? as in history? The edit tag can also be removed.  Ping @meleg.

Reply:
I do not remember that I did this change. Probably there was a typo that I fixed (wrong index for example). Clearly `err_hyst` is a typo and it should be `err_hist`.